tests/test_slow_retrieval_attack.py

```diff
@@ -157,10 +157,9 @@ def setUp(self):
     # The slow retrieval server, in mode 2 (1 byte per second), will only
     # sleep for a  total of (target file size) seconds.  Add a target file
     # that contains sufficient number of bytes to trigger a slow retrieval
-    # error.  "sufficient number of bytes" assumed to be
-    # >> 'tuf.settings.SLOW_START_GRACE_PERIOD' bytes.
+    # error.  "sufficient number of bytes" assumed to be 10x more.
     extra_bytes = 8
-    total_bytes = tuf.settings.SLOW_START_GRACE_PERIOD + extra_bytes
+    total_bytes = 100 * extra_bytes
```

Comment:
I'd be happier if re-used that variable or assigned a constant for this `100`

Reply:
@awwad, could you help me fix this? I think this might be related to the problems you've been seeing with bad hashes, and so on. Thanks in advance!